services/movie.py

```diff
@@ -0,0 +1,42 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(
+    genres_ids: list[int] = None,
+    actors_ids: list[int] = None
+) -> QuerySet:
+    queryset = Movie.objects.all()
+
+    if genres_ids:
+        queryset = queryset.filter(genres__id__in=genres_ids)
+
+    if actors_ids:
+        queryset = queryset.filter(actors__id__in=actors_ids)
+
+    return queryset
+
+
+def get_movie_by_id(movie_id: int) -> Movie:
+    return Movie.objects.all().get(id=movie_id)
+
+
+def create_movie(
+    movie_title: str,
+    movie_description: str,
+    genres_ids: list[int] = None,
```

Comment:
Use the `Optional` from `typing` or pipe operator `|` to indicate that this argument is optional in type hinting, e.g.
```suggestion
    genres_ids: list[int] | None = None,
```
Add it here and in other methods as well for correct type annotation.